chore: adaptations for nightly-2024-07-30

semorrison opened the pull request.

jcommelin reviewed (commented):
I would like to reset the whitespace changes in certain files.
```bash
git checkout bump/v4.11.0 -- Mathlib/Data/Fin/Basic.lean Mathlib/Data/Finsupp/Defs.lean
```


bors d+

semorrison commented:
> I would like to reset the whitespace changes in certain files.
> 
> ```shell
> git checkout bump/v4.11.0 -- Mathlib/Data/Fin/Basic.lean Mathlib/Data/Finsupp/Defs.lean
> ```
> 
> bors d+

As it turns out I had already fixed the whitespace on #15282. I've incorporated it here, and hopefully made it stick on `nightly-testing`.
